homeassistant/components/mqtt/climate.py

```diff
@@ -226,6 +245,13 @@ def valid_preset_mode_configuration(config: ConfigType) -> ConfigType:
         ): cv.ensure_list,
         vol.Optional(CONF_FAN_MODE_STATE_TEMPLATE): cv.template,
         vol.Optional(CONF_FAN_MODE_STATE_TOPIC): valid_subscribe_topic,
+        vol.Optional(CONF_HUMIDITY_COMMAND_TEMPLATE): cv.template,
+        vol.Optional(CONF_HUMIDITY_COMMAND_TOPIC): valid_publish_topic,
+        vol.Optional(CONF_HUMIDITY_INITIAL, default=50): cv.positive_int,
```

Comment:
Why would we need an Initial value. I would suggest assigning `None` if a state topic or command topic is set during initialisation.

Reply:
I think it is not needed to have an initial value. Initial values can be reported by the device. It might even be that I will sugest to remove those initial settings and deprecate them.
Even the `None` assignments for all attributes (https://github.com/home-assistant/core/pull/84592/files#diff-c8ebf96494fa4dca0f6c463949933572a837d982d5d9fbd8f042173bba03fbd1R466-R473) can be cleaned up since they are already initialized in the parent class. We do not need to reset those to `None` when a config update arrives.
You do not need to changed that in this PR though, I'll open a seperate one for that.